android/src/main/java/com/amplitude/android/DefaultTrackingOptions.kt

```diff
@@ -8,13 +10,10 @@ constructor(
     var deepLinks: Boolean = false,
     var screenViews: Boolean = false,
 ) {
-    var userInteractions = false
-        @ExperimentalAmplitudeFeature
-        set
-
     // Prebuilt options for easier usage
     companion object {
         @JvmField
+        @Deprecated("Use AutocaptureOptions instead", ReplaceWith("AutocaptureOptions(appLifecycles = true, deepLinks = true, screenViews = true)"))
```

Comment:
It's better to explicitly list all DET configs here even though it's true by default
```suggestion
        @Deprecated("Use AutocaptureOptions instead", ReplaceWith("AutocaptureOptions(appLifecycles = true, deepLinks = true, screenViews = true, sessions = true)"))
```

Reply:
Since `AutocaptureOption` is not an exact replacement for `DefaultTrackingOptions`, I removed `ReplaceWith`  entirely.